docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+    it:
+
+    ```yaml
+    social:
+        cards_layout: event
+    tags:
+      - events
+    ```
+
+    Now, create a post in `docs/blog/posts/events` with the following header:
+
+    ```yaml
+    ----
+    date: 2024-04-03
+    tags:
+        - material
+        - documentation
+        - made easy
+    event:
+        date: 2024-04-08
+        location: Online
+    ----
+    ```
+
+    Build and inspect the result. You should see that the correct social card
+    layout was used and that the `events` tag was added to the list of tags
+    defined in the post itself.
+
+While merging the data from the `.meta.yml` file, settings from the page header
+take precedence, so you can always override settings by adding them to the
+post itself.
+
+## 13. RSS Feeds
+
+Your readers can subscribe to a blog if you configure an RSS feed for it. An
+easy way to do this is with the [MkDocs RSS Plugin], which is will integrated
+with Material for MkDocs. Being a third-party plugin, it needs to be installed
+before it can be used.
+
+[MkDocs RSS Plugin]: https://guts.github.io/mkdocs-rss-plugin
+
+!!! example "Add an RSS feed"
+
+    Install the RSS plugin into your project:
+
+    ```
+    $ pip install mkdocs-rss-plugin
+    ```
+
+    It is important that have the `site_name`, `site_description` and
+    `site_url` settings configured as instructed in [4. Setting up your blog].
+    The RSS plugin makes use of this information to construct the feed, so make
+    sure you have configured them:
+
+    [4. Setting up your blog]: #4-setting-up-your-blog
+
+    Now, configure it in the `mkdocs.yml` in the `plugins` section.
+
+    ```yaml hl_lines="9"
+    plugins:
+        - search
+        - blog:
+            authors_profiles: true
+            categories_allowed:
+            - Holidays
+            - News
+        - tags
+        - rss
+    ```
+
+    Have a look at http://localhost:8000/feed_rss_created.xml to see the RSS
+    feed in all its XML glory. You can use a browser like Firefox or Chrome.
+
+    Unfortunately, Safari will always look for an app for RSS feeds and will not
+    display anything itself. Alternatively, you can use `curl` to get the feed
+    and `xmllint` to format it:
+
+    ```
+    curl -s http://localhost:8000/feed_rss_created.xml | xmllint --format -
+    ```
+
+    You way want to try your feed with a feed reader. There are various desktop
+    and mobile apps as well as online services. Of course, to use the latter you
+    will need to deploy your project somewhere that is accessible to them.
+
+This minimal configuration should work well if you have not made any changes
+to the default configuration of the blog plugin.
+
+The table below summarizes how the plugin constructs information for the feed
+itself and what configuration options you can use modify the defaults. Required
+RSS element are marked with an asterisk.
+
+| RSS element      | default                    | configure |
```

Comment:
Honestly, this is so much line noise and irrelevant at this point. I'm not even sure I understand it.